Aplib.Core/Desire/Goals/Goal.cs

```diff
@@ -14,15 +16,16 @@ public class Goal
         /// The abstract definition of what is means to test the Goal's heuristic function. Returns <see cref="Heuristics"/>, as
         /// they represent how close we are to matching the heuristic function, and if the goal is completed.
         /// </summary>
-        /// <seealso cref="Goal.Evaluate"/>
-        public delegate Heuristics HeuristicFunction();
+        /// <seealso cref="Goal.IsCompleted"/>
```

Comment:
```suggestion
        /// <seealso cref="IsCompleted"/>
```

This is possible right? Since it is in the same class?